knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml

```diff
@@ -957,7 +957,7 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: kafka-webhook
-  minReplicas: 1
+  minReplicas: 2
```

Comment:
Do we really want this? It'll mean users won't be able to tune this down to 1 :thinking:.

Reply:
I think I will anyways tackle this differently